src/map.js

```diff
@@ -3,13 +3,13 @@ const isPromise = require('./isPromise')
 const some = require('./some')
 const ifElse = require('./ifElse')
 const compose = require('./compose')
-const mapTransformer = require('./transformers/map')
+const mapTransformer = require('../transformers/map')
```

Comment:
Actually, I'd like to move *all* methods (internal, transformers) to the `src` folder if that's OK with you